2020-05-07 00:21:18,251 DEBUG    root            Loaded Command Group: [u'gcloud', u'projects']
2020-05-07 00:21:18,284 DEBUG    root            Loaded Command Group: [u'gcloud', u'projects', u'add_iam_policy_binding']
2020-05-07 00:21:18,287 DEBUG    root            Running [gcloud.projects.add-iam-policy-binding] with arguments: [--member: "serviceAccount:osrs.bot.rental@gmail.com@bot-farm-osrs.iam.gserviceaccount.com", --role: "roles/owner", PROJECT_ID: "YOUR_PROJECT_ID"]
2020-05-07 00:21:19,743 DEBUG    root            (gcloud.projects.add-iam-policy-binding) INVALID_ARGUMENT: Request contains an invalid argument.
Traceback (most recent call last):
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 983, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 807, in Run
    resources = command_instance.Run(args)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 533, in Run
    args, add_condition=self._add_condition)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 1014, in _GetModifiedIamPolicyAddIamBinding
    policy = self._GetIamPolicy(args)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 998, in _GetIamPolicy
    policy = get_iam_method.Call(get_iam_request)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/apis/registry.py", line 342, in Call
    return request_func(global_params=global_params)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/apis/registry.py", line 401, in RequestFunc
    return method(request, global_params=global_params)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/googlecloudsdk/third_party/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 952, in GetIamPolicy
    config, request, global_params=global_params)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 731, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 737, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
  File "/Users/jakevollkommer/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 604, in __ProcessHttpResponse
    http_response, method_config=method_config, request=request)
HttpBadRequestError: HttpError accessing <https://cloudresourcemanager.googleapis.com/v1/projects/YOUR_PROJECT_ID:getIamPolicy?alt=json>: response: <{'status': '400', 'content-length': '127', 'x-xss-protection': '0', 'x-content-type-options': 'nosniff', 'transfer-encoding': 'chunked', 'vary': 'Origin, X-Origin, Referer', 'server': 'ESF', 'server-timing': 'gfet4t7; dur=1118', '-content-encoding': 'gzip', 'cache-control': 'private', 'date': 'Wed, 06 May 2020 14:21:19 GMT', 'x-frame-options': 'SAMEORIGIN', 'alt-svc': 'h3-27=":443"; ma=2592000,h3-25=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q049=":443"; ma=2592000,h3-Q048=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"', 'content-type': 'application/json; charset=UTF-8'}>, content <{
  "error": {
    "code": 400,
    "message": "Request contains an invalid argument.",
    "status": "INVALID_ARGUMENT"
  }
}
>
2020-05-07 00:21:19,749 ERROR    root            (gcloud.projects.add-iam-policy-binding) INVALID_ARGUMENT: Request contains an invalid argument.
2020-05-07 00:21:19,768 DEBUG    root            Metrics reporting process started...
